ci/jenkins/jobs/macros.yaml

```diff
@@ -275,3 +275,55 @@
           set -e 
           DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
           ./ci/jenkins/test-mc.sh --testcase e2e --registry ${DOCKER_REGISTRY} --mc-gateway --codecov-token "${CODECOV_TOKEN}" --coverage --kind
+- builder:
```

Comment:
Better to add a new line before each `- builder:`

Reply:
New line added between each builder.